hbase/src/main/scala/akka/hbase/HBaseConverter.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.hbase
+
+import org.apache.hadoop.hbase.client.Put
+import org.apache.hadoop.hbase.util.Bytes
+
+import scala.language.implicitConversions
+
+trait HBaseConverter[T] {
+  def put(t: T): Put
+}
+
+object HBaseConverter {
+  object Implicits {
+    implicit def toBytes(string: String): Array[Byte] = Bytes.toBytes(string)
```

Comment:
I don't like this, implicit conversions are magic, I think this belongs in the user code, because it depends on how they want to transform strings to bytes (what character encoding they want to use for example).